miner/worker.go

```diff
@@ -1080,6 +1081,32 @@ func (w *worker) fillTransactions(interruptCh chan int32, env *environment, stop
 			localBlobTxs[account] = txs
 		}
 	}
+
+	if bidTxs != nil {
+		var wg sync.WaitGroup
+		wg.Add(4)
+
+		filterBidTxs := func(commonTxs map[common.Address][]*txpool.LazyTransaction) {
+			defer wg.Done()
+			for acc, txs := range commonTxs {
+				newTxs := make([]*txpool.LazyTransaction, 0)
+				for _, tx := range txs {
+					if !bidTxs.Contains(tx.Hash) {
+						newTxs = append(newTxs, tx)
```

Comment:
you can not simply remove the iteam in the slice..

What does tx conflict means:
if tx hash is same, it means the nonce of the tx is same,  then all the tx before the conflict tx from a same account is invalid. So once a conflict tx find, we need remove all the txs that is queued before this tx.

Reply:
fixed